### PR TITLE
Reformat code

### PR DESCRIPTION
**Pull request by jschlyter**
Replaces #20

### CODE REVIEW

File: pyspcwebgw/__init__.py [hunk 1]
```diff
@@ -44,10 +44,12 @@ def zones(self):
...
         """Connect websocket to SPC Web Gateway."""
-        self._websocket = AIOWSClient(loop=self._loop,
-                                      session=self._session,
-                                      url=self._ws_url,
-                                      async_callback=self._async_ws_handler)
+        self._websocket = AIOWSClient(
+            loop=self._loop,
+            session=self._session,
+            url=self._ws_url,
+            async_callback=self._async_ws_handler,
```
Comment (crallian) [inline pyspcwebgw/__init__.py:51]: extra comma sign

File: pyspcwebgw/__init__.py [hunk 2]
```diff
@@ -80,41 +81,45 @@ async def change_mode(self, area, new_mode):
...
         else:
             area_id = area
 
-        url = urljoin(self._api_url, "spc/area/{area_id}/{command}".format(
-            area_id=area_id, command=AREA_MODE_COMMAND_MAP[new_mode]))
+        url = urljoin(
+            self._api_url,
+            "spc/area/{area_id}/{command}".format(
+                area_id=area_id, command=AREA_MODE_COMMAND_MAP[new_mode]
+            ),
```
Comment (crallian) [inline pyspcwebgw/__init__.py:98]: should this comma sign be here?
98                 ),

File: pyspcwebgw/__init__.py (hunk 1 above)
Comment (jschlyter) [reply pyspcwebgw/__init__.py:51]: yes, that is how `black` formats things.

File: pyspcwebgw/__init__.py (hunk 2 above)
Comment (jschlyter) [reply pyspcwebgw/__init__.py:98]: Same here, also ok.

**Pull request merged**